mhep/mhep/dev/views/organisations.py

```diff
@@ -116,6 +117,39 @@ def delete(self, request, pk, userid):
         return Response("", status=status.HTTP_204_NO_CONTENT)
 
 
+class CreateDeleteOrganisationMembers(
+    generics.UpdateAPIView,
+):
+    serializer_class = OrganisationMemberSerializer
+    permission_classes = [
+        IsAuthenticated,
+        IsAdminOfOrganisation,
+    ]
+
+    def get_queryset(self, *args, **kwargs):
+        return getattr(self.request.user, f"{VERSION}_organisations")
+
+    def _get_user(self, userid):
+        try:
+            return User.objects.get(id=userid)
+        except User.DoesNotExist:
+            raise exceptions.NotFound(detail=f"No such user: id={userid}")
+
+    def post(self, request, pk, userid):
+        org = self.get_object()
+        user = self._get_user(userid)
+
+        org.members.add(user)
+        return Response("", status=status.HTTP_204_NO_CONTENT)
+
+    def delete(self, request, pk, userid):
+        org = self.get_object()
+        user = self._get_user(userid)
+
+        org.members.remove(user)
```

Comment:
what happened to code removing the users from `org.admins` and `org.librarians` ...?